kubernetes/staging/13-network-policy.yaml

```
@@ -0,0 +1,60 @@
+      port: 53
+  # Allow outbound HTTPS for external API calls
+  - to:
+    - ipBlock:
+        cidr: 0.0.0.0/0
+        except:
+        - 10.0.0.0/8
+        - 172.16.0.0/12
+        - 192.168.0.0/16
+    ports:
+    - protocol: TCP
+      port: 443 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)